src/peripheral/dwt.rs

```diff
@@ -318,15 +318,15 @@ impl DWT {
 /// Whether the comparator should match on read, write or read/write operations.
 #[derive(Debug, Eq, PartialEq, Copy, Clone)]
 pub enum AccessType {
-    /// Generate packet only when matched adress is read from.
+    /// Generate packet only when matched address is read from.
     ReadOnly,
-    /// Generate packet only when matched adress is written to.
+    /// Generate packet only when matched address is written to.
     WriteOnly,
-    /// Generate packet when matched adress is both read from and written to.
+    /// Generate packet when matched address is both read from and written to.
     ReadWrite,
 }
 
-/// The sequence of packet(s) that should be emitted on comparator match.
+/// The sequence of packet(s) or events that should be emitted/generated on comparator match.
 #[derive(Debug, Eq, PartialEq, Copy, Clone)]
 pub enum EmitOption {
```

Comment:
This will eventually have to be split into multiple enums, I think. Outside the scope of this PR.

Reply:
Yeah, it's a bit overloaded now but so is how ARM implemented it 😅

Reply:
I started the review thinking I had previously covered all emit options for address comparisons but that was not the case. No refactor needed at the moment.